roles/provision-unifiedpush-apb/tasks/main.yml

```diff
@@ -41,9 +41,9 @@
       - name: MYSQL_USER
         value: "{{ MYSQL_USER }}"
       - name: MYSQL_PASSWORD
-        value: "{{ MYSQL_PASSWORD }}"
+        value: "{{ mysql_password }}"
```

Comment:
why not all `UPPERCASE` ?

Reply:
My understanding is that it takes this variable https://github.com/aerogearcatalog/unifiedpush-apb/pull/11/files#diff-eca84bed92a00db089fb775737853f10R10
And if `MYSQL_PASSWORD` wouldn't be specified in the wizard, it would be undefined.

Reply:
Ah, ignore my previous comment, now I know what you mean *facepalm_with_both_hands*